generate_data.py

```diff
@@ -171,11 +171,17 @@ def main():
         args.max_seq_len -= biobert.num_special_tokens_to_add()
         tokenizer = biobert.tokenize
         is_bert_tokenizer = True
+        
+    elif args.tokenizer == 'spacy_lg':
+        import spacy
+        tokenizer = spacy.load("en_core_web_lg")
```

Comment:
Bug: This line gives me an error `[E050] Can't find model 'en_core_web_lg'.`
We might need to add a link in `requirements.txt` similar to the one [here](https://github.com/smitkiri/ehr-relation-extraction/blob/1ae6b6a8d37498c16ed333d061b7ec9f9494c1eb/requirements.txt#L3) so that this model is downloaded when we do `pip install -r requirements.txt` when setting up the repo.
Since this model will be downloaded with `pip`, we'll need to change how we load the model ([as mentioned here](https://github.com/smitkiri/ehr-relation-extraction/blob/1ae6b6a8d37498c16ed333d061b7ec9f9494c1eb/requirements.txt#L3)).

Another option might be to download the model here somehow if it doesn't exist, so we don't download too many things when setting up the repo.

Reply:
Have added spacy model to the req.txt. This downloads the spacy_lg model for me!